### ISSUE TITLE
anima 的训练预浏览图出图问题

### ISSUE THREAD

**Issue by rongzi5** [WhitecrowAurora/lora-rescripts#15]
<img width="326" height="220" alt="Image" src="https://github.com/user-attachments/assets/7d7a2d71-6347-4fcd-a0f0-11506416651c" />

在使用 Anima 框架进行 LoRA 模型训练时，训练过程中生成的预览图出现异常

附上配置。配置为默认配置，只修改了模型加载和数据集部分
```toml
model_train_type = "anima-lora"
pretrained_model_name_or_path = "D:/anima/anima-preview3-base.safetensors"
vae = "D:/anima/qwen_image_vae.safetensors"
qwen3 = "D:/anima/qwen_3_06b_base.safetensors"
qwen3_max_token_length = 512
t5_max_token_length = 512
timestep_sampling = "shift"
sigmoid_scale = 1
discrete_flow_shift = 3
weighting_scheme = "uniform"
attn_mode = ""
split_attn = false
vae_disable_cache = false
unsloth_offload_checkpointing = false
train_data_dir = "C:/test"
prior_loss_weight = 1
resolution = "1024,1024"
enable_bucket = true
min_bucket_reso = 256
max_bucket_reso = 2048
bucket_reso_steps = 64
bucket_no_upscale = true
bucket_selection_mode = "legacy"
output_name = "aki"
output_dir = "./output"
save_model_as = "safetensors"
save_precision = "fp16"
save_every_n_epochs = 2
save_state = false
save_state_on_train_end = false
max_train_epochs = 10
train_batch_size = 1
gradient_checkpointing = true
gradient_accumulation_steps = 1
network_train_unet_only = true
network_train_text_encoder_only = false
peak_vram_control_enabled = false
enable_mixed_resolution_training = false
learning_rate = 0.0001
unet_lr = 0.0001
text_encoder_lr = 0.00001
lr_scheduler = "cosine_with_restarts"
lr_scheduler_args = [ ]
lr_warmup_steps = 0
lr_scheduler_num_cycles = 1
optimizer_type = "AdamW8bit"
lora_type = "lora"
network_dim = 16
network_alpha = 16
dim_from_weights = false
network_module = "networks.lora_anima"
pissa_init = false
lulynx_experimental_core_enabled = false
randomly_choice_prompt = false
positive_prompts = "newest, safe, 1girl, masterpiece, best quality"
negative_prompts = ""
sample_width = 1024
sample_height = 1024
sample_cfg = 4
sample_seed = 42
sample_steps = 25
sample_sampler = "euler"
sample_scheduler = "simple"
sample_every_n_epochs = 1
sample_at_first = true
log_with = "tensorboard"
logging_dir = "./logs"
validation_split = 0
caption_extension = ".txt"
shuffle_caption = false
keep_tokens = 0
caption_tag_dropout_target_mode = "drop_all"
caption_tag_dropout_target_count = 1
prefer_json_caption = true
json_caption_hint = """
推荐 JSON 结构顺序：quality / count / character / series / artist / appearance[] / tags[] / environment[] / nl。
当启用 shuffle_caption 时，只会打乱 appearance / tags / environment 三组内部顺序，前面的固定字段顺序会保留。"""
noise_offset_random_strength = false
ip_noise_gamma_random_strength = false
seed = 1337
clip_skip = 2
masked_loss = false
alpha_mask = false
ema_enabled = false
safeguard_enabled = false
no_metadata = false
skip_until_initial_step = false
mixed_precision = "bf16"
torch_compile = false
dynamo_backend = "inductor"
opt_channels_last = false
lowram = false
cache_latents = true
cache_latents_to_disk = true
cache_text_encoder_outputs = true
cache_text_encoder_outputs_to_disk = true
persistent_data_loader_workers = true
cpu_offload_checkpointing = false
pytorch_cuda_expandable_segments = true
fp8_base = false
fp8_base_unet = false
disable_mmap_load_safetensors = false
cooldown_poll_seconds = 15
enable_debug_options = false
enable_distributed_training = false
num_machines = 1
machine_rank = 0
main_process_port = 29500
sync_config_from_main = true
sync_config_keys_from_main = "*"
sync_missing_assets_from_main = true
sync_asset_keys = "pretrained_model_name_or_path,train_data_dir,reg_data_dir,vae,resume"
sync_main_toml = "./config/autosave/distributed-main-latest.toml"
sync_ssh_port = 22
sync_use_password_auth = false
clear_dataset_npz_before_train = false
network_args = [ "anima_adapter_type=lora" ]

**Comment by rongzi5** [comment]
[run_gui-20260419-221854.log](https://github.com/user-attachments/files/26882235/run_gui-20260419-221854.log)

**Comment by shira-qwq** [comment]
这个比较老的版本好像也有出过bug，5天前的版本预览图我用过是好的

**Comment by WhitecrowAurora** [comment]
请问版本号是多少?

**Comment by rongzi5** [comment]
> 请问版本号是多少?

lora-rescripts-v1.4.4-Beta34

**Comment by WhitecrowAurora** [comment]
我看一下

**Comment by WhitecrowAurora** [comment]
已在beta36中补全修复,如果还有问题请尝试打开debugmode把输出复制给我
